pennylane/interfaces/jax_jit_tuple.py

```diff
@@ -231,13 +206,19 @@ def wrapper(p):
         res = jax.pure_callback(wrapper, shape_dtype_structs, params, vectorized=True)
         return res
 
-    @execute_wrapper.defjvp
+    @partial(execute_wrapper.defjvp, symbolic_zeros=True)
```

Comment:
Can't find any useful documentation on this. What does passing in `symbolic_zeros=True` actually do?

Reply:
The symbolic zeros mark the non-trainable tangents as zeros, it does not use any memory and is the solution Jax team told me to use to identify trainable parameters. Unfortunately documentation is as you said non-existent.